### DISCUSSION TITLE
How to "hide" a window instead of closing it

### DISCUSSION THREAD

**Discussion by Xiaobaishushu25** [slint-ui/slint#2828] [Q&A] [answered]
Hello, I want to use slint to create a small tool. My goal is to hide the window when it is not at the top of the desktop or when I click the close button, and then show it when I press a shortcut key. However, when I use app. window(). hide() to hide the window, the program exits directly after the window closes. So, how can I achieve my goal? Thank you for your patient reading.

**Comment by ogoffart** [comment, marked as answer]
This is currently only possible using private API
This is being tracked on https://github.com/slint-ui/slint/issues/1499
